src/seedu/addressbook/AddressBook.java

```diff
@@ -1164,7 +1164,7 @@ private static String getUsageInfoForExitCommand() {
      * @return  string without the prefix
      */
     private static String removePrefix(String fullString, String prefix) {
-        return fullString.replace(prefix, EMPTY_STRING);
+        return fullString.replaceFirst(prefix, EMPTY_STRING);
```

Comment:
This revision still does not remove the value of prefix , from the beginning of the 'fullString' variable. 
The method should replace the value only if the value of prefix is a prefix of the value of fullString.